frontend/src/tests/components/commons/SystemMessage.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { mount } from "enzyme";
+import SystemMessage, {
+  MESSAGE_TYPE,
+  UNIT_TEST_VALIDATION
+} from "../../../components/commons/SystemMessage";
+
+it("renders warning message", () => {
+  const wrapper = mount(
+    <SystemMessage messageType={MESSAGE_TYPE.warning} title={"title"} message={"message"} />
+  );
+  const msgTypeValidation = UNIT_TEST_VALIDATION.validation;
+  const title = <h3>title</h3>;
+  const message = <p>message</p>;
+  expect(msgTypeValidation === MESSAGE_TYPE.warning).toEqual(true);
```

Comment:
I see why you did this, but we could still have the icons render incorrectly and have this test pass (we even switched it up yesterday). 

In this case, I think it's totally appropriate to check if the wrapper contains the right icon, `<i class="fas fa-exclamation-circle" style="font-size:40px"></i>`
Since the logic in our component is to build the icon correctly.

Reply:
Added this validation in commit #https://github.com/code-for-canada/project-thundercat/pull/125/commits/c87a4ae6ec61eda17eb3d71450e3476f23f0c11b

Reply:
Great! Now that this is tested you no longer need any of the `UNIT_TEST_VALIDATION` code.

Reply:
I simplified the icon and alert classes tests.
See commit #https://github.com/code-for-canada/project-thundercat/pull/125/commits/f21b4a83c0d58c11b5e055a04d939a368dd15e74